### DISCUSSION TITLE
How to Reload compile_commands.json

### DISCUSSION THREAD

**Discussion by Clinkz24** [microsoft/vscode-cpptools#10001] [Q&A] [answered]
I need to restart vscode after update compile_commands.json,If not , IntelliSense will not work for new file.
it is some way to make IntellSense auto load compile_commands.json?

**Comment by sean-mcmanus** [comment, marked as answer]
Are you hitting https://github.com/microsoft/vscode-cpptools/issues/7610 ? My guess is that you are. Does it work for you after you change the compileCommands property to a non-relative path?

You could also do a Reload Window.
